Disable PDF.js built-in findbar

nt1m opened the pull request:
#### cc7b06259688a71f2067e4fbabbbed0ec039781f
<pre>
Disable PDF.js built-in findbar
<a href="https://bugs.webkit.org/show_bug.cgi?id=237045">https://bugs.webkit.org/show_bug.cgi?id=237045</a>
&lt;rdar://89301856&gt;

Reviewed by Brent Fulgham.

* Source/WebCore/Modules/pdfjs-extras/cocoa/style.css:
(#findbar):
(#findbar:not(.hidden) ~ #viewerContainer): Deleted.
(#findbar::before,): Deleted.
(#findbarOptionsTwoContainer): Deleted.
(#findResultsCount,): Deleted.
(#findbar .splitToolbarButton &gt; .toolbarButton::before): Deleted.
(#findInput:focus): Deleted.
(#findInput[data-status=&quot;notFound&quot;]): Deleted.
* Source/WebCore/Modules/pdfjs-extras/content-script.js:
(const.PDFJSContentScript.overrideSettings):
(const.PDFJSContentScript.init):

Canonical link: <a href="https://commits.webkit.org/253226@main">https://commits.webkit.org/253226@main</a>
</pre>

brentfulgham commented on Source/WebCore/Modules/pdfjs-extras/cocoa/style.css:
```diff
@@ -185,66 +185,8 @@ body {
-#findInput[data-status="notFound"] {
-    background: var(--field-bg-color);
-    color: var(--field-color);
-}
```
Deleted code is the best code! :-D

brentfulgham reviewed (approved):
LGTM

nt1m committed:
Disable PDF.js built-in findbar
https://bugs.webkit.org/show_bug.cgi?id=237045
<rdar://89301856>

Reviewed by Brent Fulgham.

* Source/WebCore/Modules/pdfjs-extras/cocoa/style.css:
(#findbar):
(#findbar:not(.hidden) ~ #viewerContainer): Deleted.
(#findbar::before,): Deleted.
(#findbarOptionsTwoContainer): Deleted.
(#findResultsCount,): Deleted.
(#findbar .splitToolbarButton > .toolbarButton::before): Deleted.
(#findInput:focus): Deleted.
(#findInput[data-status="notFound"]): Deleted.
* Source/WebCore/Modules/pdfjs-extras/content-script.js:
(const.PDFJSContentScript.overrideSettings):
(const.PDFJSContentScript.init):

Canonical link: https://commits.webkit.org/253226@main
